gamestates/cSelectYourNextConquestState.cpp

```diff
@@ -111,10 +111,8 @@ void cSelectYourNextConquestState::thinkFast() {
         }
 
         cMessageDrawer *pDrawer = drawManager->getMessageDrawer();
-        char *cMessage = pDrawer->getMessage();
-
         isFinishedConqueringRegions = true;
-        bool isNotDisplayingMessage = cMessage[0] == '\0';
+        bool isNotDisplayingMessage = pDrawer->hasMessage();
```

Comment:
Logic inverterted here accidentally.
```suggestion
        bool isNotDisplayingMessage = !pDrawer->hasMessage();
```
or perhaps
```suggestion
        bool isDisplayingMessage = pDrawer->hasMessage();
```
And then invert the logic that uses the boolean.